Responding to issues 19

csKyou opened the pull request:
close #19

csKyou committed:
Responding to issues 19
・Functionize the document.body of the CellManager and move it to js

csKyou committed:
Responding to issues 19

dixonsiu commented on src/js/login.js:
```diff
@@ -135,6 +247,22 @@ login.getToken = function(unitCellUrl, loginInfo) {
     });
 }
 
+login.refreshToken = function(unitCellUrl, refreshInfo) {
```
Shouldn't unitCellUrl be cellUrl because it can be the Unit Cell or normal Cell?

csKyou replied:
Argument name changed.
unitCellUrl -> cellUrl

dixonsiu commented on bar/90_contents/src/login.html:
```diff
@@ -24,88 +24,7 @@
-                $("#unitUrl").val(cellInfo.rootURL);
-                $("#unitCellName").val(cellInfo.cellName);
-            }
+            uLogin.initCellManager();
```
Do you need to perform the same changes to the normal login.html, too?

csKyou replied:
I agree.
Passing arguments to initCellManager makes it possible to do it, and I think that it is unnecessary to split html into two.

csKyou replied:
Since processing to save token is a function in js, it is not necessary to modify login.html of unitCell.
However, since processing in document.ready is a function, I think that it is possible to share it by adding arguments.

dixonsiu reviewed (commented):
Please check my questions.

csKyou committed:
Responding to issues 19
・Review compliance

csKyou committed:
Responding to issues 19
・When UnitManager was in operation, we fixed it because token information was not added.
